docs/redirects.asciidoc

```diff
@@ -193,3 +193,106 @@ This page has moved. Refer to <<url_templating-language>>.
 === Variables
 
 This page has moved. Refer to <<url-template-variables>>.
+
+[float]
+[[time-series-visual-builder]]
+=== Time Series Visual Builder
+
+This page was deleted. Please see <<Dashboard>>.
+
+[float]
+[[development-functional-tests]]
```

Comment:
This is in the docs - https://www.elastic.co/guide/en/kibana/current/development-tests.html#development-functional-tests. You can remove lines 203-207.